src/__tests__/Input.spec.js

```diff
@@ -1,272 +1,250 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import { Input } from '..';
+import { testForDefaultTag } from '../testUtils';
 
 describe('Input', () => {
   it('should render with "input" tag when no type is provided', () => {
-    const wrapper = shallow(<Input />);
-
-    expect(wrapper.type()).toBe('input');
+    testForDefaultTag(Input, 'input');
   });
 
   it('should render with "type" tag when type is "select"', () => {
-    const wrapper = shallow(<Input type="select">Yo!</Input>);
-
-    expect(wrapper.type()).toBe('select');
+    render(<Input type="select">Yo!</Input>);
+    expect(screen.getByText(/yo/i).tagName.toLowerCase()).toMatch('select');
```

Comment:
You can also check if `document.querySelector('select')` exists. Or more specifically you can do `container.querySelector` where `container` is from `const { container } = render(...)`